src/Hyperion/Main.hs

```diff
@@ -78,7 +81,8 @@ options = do
          (Options.strOption
             (Options.long "output" <>
              Options.short 'o' <>
-             Options.metavar "FILENAME"))
+             Options.help "Outputs the benchark to the directory" <>
```

Comment:
> thus providing a file would have them overwrite each other, as each benchmark executable would try to write to the same file.

That's true, but it's also not our problem. We shouldn't be assuming the user will use Stack, or indeed even `cabal-install`. And the common case really is that the user has one project, containing one package, with one benchmark suite. Benchmarks are just binaries like any other, so we should make the common case easy, the general case possible.

I don't understand what the diff is between your proposed behaviour and the one I wrote above. Aren't they precisely the same?